README.md

```diff
@@ -166,7 +166,7 @@ mode or the command line mode.
 
 There are many bugs and TODOs.
 
-Plotman will always look in the current directory for the `config.yaml` file
+Plotman will always look in the current directory for the `plotman.yaml` file
```

Comment:
Is this still correct or is that outdated, now that `generate` actually drops the config file into the config directory defined by `appdirs`?

Reply:
I updated the README to be accurate. Thanks for the catch!